css/style.css

```diff
@@ -17,3 +13,168 @@ body {
  */
 
 
+ body {
+    font-family: 'Roboto', sans-serif;
+    -webkit-font-smoothing: antialiased;
+}
+
+ul {
+    list-style: none;
+}
+
+ /**************************************** Styling navigation bar ****************************************************************/
+ 
+.nav-bar {
+    background-color: #fff;
+    position: fixed;
+    width: 100%;
+    border-bottom: 1px solid #DDD;
+    display: block;
+}
```

Comment:
Its best to leave an extra new line after each closing bracket of a css statement to make the css more readable and easier to scan through when you revisit it in the future.